src/components/PlaylistModal/PlaylistModal.css

```diff
@@ -0,0 +1,54 @@
+.modal{
+    width: 25rem;
+    height: 13rem;
+}
+
+.modal-wrapper{
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    background-color: rgb(234, 234, 236,0.3);
+    z-index: 15;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.modal-wrapper .modal{
+    border: 1px solid var(--lynx-gray-color);
+}
+
+.playlist-modal{
+    background-color: var(--primary-color-50);
+    border: 1px solid var(--lynx-gray-color);
+    width: 15rem;
+    padding: 1rem;
+    border-radius: var(--border-radius);
+}
+
+.playlist-modal button{
+    float: right;
+}
+
+.playlist-modal button.btn{
+    float: none;
+    margin: 1rem auto;
+}
+
+.playlist-modal ul{
+    list-style: none;
+    margin: 0.5rem 0;
```

Comment:
Suggestion : For specific margins and paddings make some utility class have them in the global scope and use them . 
This will help you in maintaining the linearity in UI and in code too

Reply:
Thanks for the suggestion, will try adding them to the component library.